test/data/schema.js

```diff
@@ -0,0 +1,317 @@
+import {
+  GraphQLBoolean,
+  GraphQLID,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLSchema,
+  GraphQLString,
+} from 'graphql';
+import {
+  connectionArgs,
+  connectionDefinitions,
+  connectionFromArray,
+  cursorForObjectInConnection,
+  fromGlobalId,
+  globalIdField,
+  mutationWithClientMutationId,
+  nodeDefinitions,
+  toGlobalId,
+} from 'graphql-relay';
+import { subscriptionWithClientId } from 'graphql-relay-subscription';
+
+import {
+  Todo,
+  User,
+  addTodo,
+  changeTodoStatus,
+  getTodo,
+  getTodos,
+  getUser,
+  getViewer,
+  markAllTodos,
+  removeCompletedTodos,
+  removeTodo,
+  renameTodo,
+} from './database';
+
+/* eslint-disable no-use-before-define */
+
+const { nodeInterface, nodeField } = nodeDefinitions(
```

Comment:
curious, did you really need relay here? it might just simplify test logic without it

Reply:
it was harder to remove it. I just wanted to get something working that reproduced the issue